crates/daphne/src/protocol/aggregator.rs

```diff
@@ -398,7 +411,16 @@ impl DapTaskConfig {
             let mut processed = HashSet::with_capacity(report_count_hint);
             let mut reports = pin!(reports);
             while let Some(report) = reports.next().await {
-                if processed.contains(&report.report_metadata.id) {
+                let enforce_replay_check = {
+                    #[cfg(feature = "report-generator")]
+                    {
+                        use std::sync::atomic::Ordering;
```

Comment:
Since we're using a setter for the atomic: https://github.com/cloudflare/daphne/pull/621/files#diff-5bbd4da852a4404371d706296eb9269892f0d85ded2d6570bcc7de42d6c82c35R60
we could consider adding a getter to clean up usage here.